### PR TITLE
#3 create top image component with the button search

### PR DESCRIPTION
**Pull request by DanaEFalah**
#3

### CODE REVIEW

**Comment by Karyum** [comment]
can we have the issue linked in the description please ?

File: package.json [hunk 1]
```diff
@@ -54,6 +54,7 @@
         "@babel/core": "^7.0.1",
         "@babel/register": "^7.0.0",
         "@loadable/component": "^5.10.3",
+        "@material-ui/icons": "^4.5.1",
```
Comment (Karyum) [inline package.json]: are you using this ?

File: src/modules/common/components/Button/index.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './style.scss';
+
+class Button extends React.Component {
```
Comment (Karyum) [inline src/modules/common/components/Button/index.jsx]: should give this a different name like SearchButton ? cause it's not just a button, the name is too general.

File: src/modules/common/components/Form/index.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './style.scss';
+
+
+class Form extends React.Component {
```
Comment (Karyum) [inline src/modules/common/components/Form/index.jsx]: what is going on this component ? also again the name is too general

File: src/modules/common/components/Image/index.jsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './style.scss';
+import Button from '../Button';
+
+class Image extends React.Component {
```
Comment (Karyum) [inline src/modules/common/components/Image/index.jsx]: again the name is too general, and also not correct

**Comment by Karyum** [comment]
@kira-abl nevermind i guess :sweat_smile:

**Comment by kira-abl** [comment]
That was lovely. I was just going over it. @Karyum @faris114

**Comment by faris114** [comment]
I am sorry @kira-abl  for faster workflow one of us have to merge asap let's try to clean all PR before swapping teams

**Pull request merged**